sdks/java/build-tools/pom.xml

```diff
@@ -0,0 +1,19 @@
+<project>
```

Comment:
Can you follow the style of other `pom.xml`s?

- Copyright header is missing.
- Remove the current comment; not particularly informative.
- Some tags missing, such as ?xml and others.